R/createSplitGate.R

```diff
@@ -0,0 +1,79 @@
+#' Create split gate
+#'
+#' Creates a split gate. split gates have two sectors (right and left),
+#' each with a unique gid and name.
+#'
+#' @param experimentId The ID of the experiment to which to add the gate.
+#' @param xChannel The name of the x channel to which the gate applies.
+#' @param name The name of the gate. Each sector is named with this parameter
+#' and a split flag (L, R).
+#' @param x The x coordinate of the center point (after the channel's scale has been applied).
+#' @param y The y coordinate of the dashed line extending from the center point (after the channel's scale has been applied).
+#' @param gid Top-level group ID of the gate, used for tailoring. If this is not specified,
+#'   then a new Group ID will be created. For compound gates, "gid" refers to the
+#'   top-level GID. Each sector has a unique model.gid and name to which
+#'   populations must refer.
+#' @param gids Group IDs of each sector, assigned to model.gid.
+#' @param parentPopulationId ID of the parent population. Use \code{NULL} for
+#'   the "ungated" population. If specified, do not specify \code{parentPopulation}.
+#' @param parentPopulation Name of the parent population. An attempt will be made
+#'   to find the population by name. If zero or more than one population exists
+#'   with the name, an error will be thrown. If specified, do not specify
+#'   \code{parentPopulationId}.
+#' @param tailoredPerFile Whether this gate is tailored per FCS file.
+#' @param fcsFileId ID of FCS file, if tailored per file. Use \code{NULL} for
+#'   the global gate in a tailored gate group. If specified, do not specify
+#'   \code{fcsFile}.
+#' @param fcsFile Name of FCS file, if tailored per file. An attempt will be made
+#'   to find the file by name. If zero or more than one file exists with the name,
+#'   an error will be thrown. If specified, do not specify \code{fcsFileId}.
+#' @param locked Prevents modification of the gate via the web interface.
+#' @param createPopulation Automatically create corresponding population.
+#' @export
+#' @examples
+#' \dontrun{
+#' createSplitGate(experimentId, "FSC-A", "my gate", 144000, 100000)
+#' }
+createSplitGate = function(experimentId, xChannel, name, x, y,
```

Comment:
This and `createQuadrantGate` both have warnings in CI (in the Check step) about doc mismatches.

Reply:
Updated and passed

Reply:
Looks like a few more still:

```
Undocumented arguments in documentation object 'createQuadrantGate'
  ‘labels’
Documented arguments not in \usage in documentation object 'createQuadrantGate':
  ‘angles’

Undocumented arguments in documentation object 'createSplitGate'
  ‘labels’ ‘skewable’
```